plugins/1password/1password.plugin.zsh

```diff
@@ -0,0 +1,33 @@
+# opswd puts the password of the named service into the clipboard. If there's a
+# one time password, it will be copied into the clipboard after 5 seconds. The
+# clipboard is cleared after another 10 seconds.
+function opswd() {
```

Comment:
Was about to write this plugin when I saw you had it here already. One change that would be good would be to add completion loading into the plugin. So, at the top of the file:

```
if type op; then
     eval "$(op completion zsh)"; compdef _op op
fi
```